Changelog.md

```diff
@@ -8,6 +8,7 @@ Compiler Features:
  * SMTChecker: Support inherited state variables.
  * SMTChecker: Support tuple assignments and function calls with multiple return values.
  * SMTChecker: Support ``delete``.
+ * SMTChecker: Inline external function calls to ``this``.
```

Comment:
Maybe add `Support` like the other sentences

Reply:
I think support is a different thing, meaning something that is standard. `Inline (...)` is more like a behavior choice